README.md

```diff
@@ -100,7 +100,7 @@ commands:
 
 And:
 
-	docker container create --name medusajail nginx # feel free to pick any image
+	docker container run --name medusajail nginx # feel free to pick any image
```

Comment:
the original command line is correct and it will create a container, please revert this change

Reply:
Hello, will do, I just want to double check with you please, 

It will create the container but not start it.

Here is the output of a telnet HP if create is used

```
[admin@TNAS ~]$ ls
Error response from daemon: Container 0c8590cd3ea557ac4d9dc112efb208cf3ce65daab682ba0dc6334631daeaaed4 is not running
[admin@TNAS ~]$ 
```

Reply:
yes that is correct, you have to start it manually... i have just changed the README to reflect this information, does it make more sense now?